setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+
+
+def readme():
+    with open('README.md') as f:
+        return f.read()
+
+setup(name='deep_qa',
+      version='0.1',
+      description='Using deep learning to answer Aristo\'s science questions',
+      long_description=readme(),
+      classifiers=[
+          'Development Status :: 3 - Alpha',
+          'License :: OSI Approved :: Apache Software License',
+          'Programming Language :: Python :: 3.5',
+          'Topic :: Scientific/Engineering :: Artificial Intelligence',
+      ],
+      keywords='deep_qa NLP deep learning machine reading',
+      url='https://github.com/allenai/deep_qa',
+      author='Matt Gardner',
+      author_email='mattg@allenai.org',
+      license='Apache',
+      packages=['deep_qa'],
+      install_requires=[
```

Comment:
oh right because there are issues with tensorflow; yeah this is fine i think. lots of the stuff in requirements.txt is only really necessary for development